.prettierrc.cjs

```diff
@@ -18,6 +18,7 @@ const config = {
   ],
   importOrderSeparation: true,
   importOrderSortSpecifiers: true,
+  trailingComma: 'all',
```

Comment:
Can you move this to below `printWidth`?

Reply:
done